operations/mimir/autoscaling.libsonnet

```diff
@@ -256,8 +256,8 @@
         // To scale out relatively quickly, but scale in slower, we look at the max memory utilization across all distributors over 15m.
         query: 'max_over_time(sum(container_memory_working_set_bytes{container="%s",namespace="%s"})[15m:])' % [name, $._config.namespace],
 
-        // threshold is expected to be a string, so use '' to cast any ints returned by siToBytes.
-        threshold: siToBytes(distributor_memory_requests) + '',
+        // threshold is expected to be a string
+        threshold: std.toString(siToBytes(distributor_memory_requests)),
```

Comment:
Have you considered rounding the number in `siToBytes`? I think returning a number of bytes with decimal doesn't make much sense.

Reply:
@pracucci true, but I thought it would perhaps be the most robust to round down to an integer on line 85 (see comment). WDYT of this approach?

Reply:
We could do both since there are other functions/autoscalers that will set a threshold.

I agree that returning bytes with a decimal doesn't make sense.

Reply:
@jhesketh sounds good, I can also try to fix siToBytes.